src/Charcoal/Cache/ServiceProvider/CacheServiceProvider.php

```diff
@@ -267,6 +267,17 @@ public function registerService(Container $container)
 
             return new CachePoolFacade($args);
         };
+
+        /**
+         * Provides a way to add custom processing over the cache key for the cache middleware.
+         *
+         * @return Closure
+         */
+        $container['cache/process-cache-key-callback'] = function () {
+            return function ($key) {
+                return $key;
+            };
+        };
```

Comment:
Appears to be obsolete.

Reply:
Indeed, je croyais l'avoir enlevé thanks